trikot-viewmodels-declarative-flow/swift/swiftui/Components/Snackbar/VMDSnackBarStateObject.swift

```diff
@@ -11,16 +11,16 @@ public class VMDSnackBarStateObject: ObservableObject {
 
     public func startWatching(_ flow: VMDFlow<VMDSnackbarViewData>) {
         flow.watch { [weak self] viewData, closeable in
-            self?.message = viewData.message
```

Comment:
I think we should do a guard here instead.

Reply:
If we do a guard and early exit, the closeableExecuter may not be set and we may never stop subscribing to changes.

Reply:
How about we guard but still unsubscribe?
Receiving null should never happen as on kotlin side the generic is typed non-null but it'd be better to just skipping the event than showing an empty snackbar.

Reply:
We will:

- Assign the closeableExecutor value as soon as possible
- After that assignment, early exit if the view data is nil